Pepperdash Core/Pepperdash Core/Comm/CommunicationStreamDebugging.cs

```diff
@@ -81,9 +81,14 @@ public void SetDebuggingWithSpecificTimeout(eStreamDebuggingSetting setting, uin
 
             if (DebugExpiryPeriod != null)
```

Comment:
These lines can be replaced with a call to `DisableDebugging`, instead of duplicating what `DisableDebugging` is doing here.

Reply:
DisableDebugging has an extra command to actually disable debugging, though. So doing that will disable debugging then immediately enable it again. It was a little confusing to me for the new command to always call DisableDebugging even if it wasn't a disable command. I agree it can be cleaned up better.

Reply:
Let's do this: Add a method called `ResetDebugTimer` or something similar. Put JUST the timer reset logic in there, then call that from both DisableDebugging and this method, where the timer needs to be reset.